.github/workflows/CI.yaml

```
@@ -195,11 +195,20 @@ jobs:
     needs: prepare_ci_run
     uses: ./.github/workflows/component-test.yml
 
-  integration_tests:
+  integration_tests_helm:
     name: Integration Tests
     needs: [prepare_ci_run, build_image]
     with:
       runtime_tag: dev-${{ needs.prepare_ci_run.outputs.DATETIME }}
+      helm-install: true
+    uses: ./.github/workflows/integration-test.yml
+
+  integration_tests_manifest:
+    name: Integration Tests
+    needs: [ prepare_ci_run, build_image ]
+    with:
+      runtime_tag: dev-${{ needs.prepare_ci_run.outputs.DATETIME }}
+      helm-install: false
```

I'm not 100% sure this works, but you could try doing the same with a job matrix:
```suggestion
  integration_tests:
    name: Integration Tests
    needs:
      - prepare_ci_run
      - build_image
    strategy:
      matrix:
        helm-install:
          - "true"
          - "false"
    with:
      runtime_tag: dev-${{ needs.prepare_ci_run.outputs.DATETIME }}
      helm-install: ${{ matrix.helm-install }}
    uses: ./.github/workflows/integration-test.yml
```